package.json

```diff
@@ -14,7 +14,11 @@
     },
     "files": [
         "/contracts",
-        "/types"
+        "/types",
+        "!contracts/drafts/",
+        "!contracts/examples/",
```

Comment:
This is wrong, `contracts/examples` and `contracts/test/` should not be ignored.

Reply:
And neither `contracts/drafts` for that matter.

Reply:
And you could also let the `Migrations.sol` in there, in case the user wants to spin up a version of the package themselves.

Reply:
No, because the user will have the migrations in his own project. Ours is pointless to him.
Regarding `contracts/examples` I think they are only examples, shouldn't be packed. Aren't `contracts/test/` used just for tests?

Reply:
Yes, `contracts/tests` are used for tests, but since the user might check the test to see how to use the contract, I though he might need those. `contracts/examples` is definitely needed, EnergyMarket.sol is there, for example.

Reply:
We need to make a distinction between the repository and the package, and maybe eventually even split things into two repositories.

**Package**

Developers would only use the package if they intend to reuse code. The contracts in `examples` are not to be reused. They are there to teach developers how to do things. Maybe they could be reused as a base, but we are not really coding them that way.

The contracts in `contracts/test` are not to be reused either in a package.

In reality, the contents of the package will be a small fraction of the contracts in the repository. We might spin these off to a different hq20 repo, and offer them to openzeppelin if they want them.

**Repository**

The contents of the repository are a different matter. Developers would come to the repository to learn how to do stuff, and for that they would use all the resources including `examples`, `test` and `contracts/test`. However, developers can get this content either with `git` or through github.

Some of the coolest stuff in the repo (uniswap exchange, energy market, the DAO you are working on) I wouldn't include it in the package. Maybe at some point we will do.

Reply:
Got it!